File: dotnet/01.EstruturaSequencial/Ex05.cs
namespace EstruturaSequencial;

static class Ex05
{
    /// <summary>
    /// Faça um Programa que converta metros para centímetros.
    /// </summary>
    public static void Run()
    {
        Console.Write("Metros: ");
        float metros = float.Parse(Console.ReadLine()!.Replace(".", ","));
        float centimetros = metros * 100;
        Console.Write($"{metros:F2}m convertido em centímetros fica {centimetros:F2}cm");
    }
}

File: nodejs/01-estrutura-sequencial/ex05.js
/*
 * Exercício 05
 *
 * Faça um Programa que converta metros para centímetros.
 */

import * as readlinePromises from 'node:readline/promises'
import {stdin as input, stdout as output} from 'node:process'

const rlp = readlinePromises.createInterface({input, output})

const metros = parseFloat(await rlp.question("Metros: "))
rlp.close()
const centimetros = metros * 100
console.log(`${metros.toFixed(2)}m convertido em centímetros fica ${centimetros.toFixed(2)}cm`)
